applications/connecting_applications_to_services/getting-started-with-service-binding.adoc

```diff
@@ -0,0 +1,37 @@
+[id="getting-started-with-service-binding"]
+= Getting started with service binding
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: getting-started-with-service-binding
+
+toc::[]
+
+{servicebinding-title} manages the data plane for workloads and backing services. This guide provides instructions with examples to help you create a database instance, deploy an application and use {servicebinding-title} to create a binding connection between the application and the database service.
```

Comment:
```suggestion
{servicebinding-title} manages the data plane for workloads and backing services. This guide provides instructions with examples to help you create a database instance, deploy an application, and use {servicebinding-title} to create a binding connection between the application and the database service.
```
Per "Commas" section of IBM Style Guide, "Use commas to separate items in a series of three or more. Use a comma before the conjunction that precedes the final item."